tools/issue-labeler/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/go-yaml/yaml"
+	"github.com/golang/glog"
+)
+
+func main() {
+	file, err := os.ReadFile("enrolled_teams.yaml")
+	if err != nil {
+		glog.Exitf("Error reading enrolled teams yaml: %v", err)
+	}
+	enrolledTeams := make(map[string][]string)
+	err = yaml.Unmarshal(file, &enrolledTeams)
+	if err != nil {
+		glog.Exitf("Error unmarshalling enrolled teams yaml: %v", err)
+	}
+	issueBody := os.Getenv("ISSUE_BODY")
+	sectionRegexp := regexp.MustCompile(`### (New or )?Affected Resource\(s\)[^#]+`)
+	affectedResources := sectionRegexp.FindString(issueBody)
+	var labels []string
+	for label, resources := range enrolledTeams {
+		for _, resource := range resources {
+			if strings.Contains(affectedResources, resource) {
```

Comment:
It might be worth logging which resource was found at the time this was run, for debugging purposes

Reply:
Done.